bitbucket-pipelines.yml

```diff
@@ -1,17 +1,122 @@
+# These are yaml anchors, reused later
+x-bash-env-steps:
+  - &bash_env_export export BASH_ENV="$BITBUCKET_CLONE_DIR/.bashrc"
+  - &bash_env_source source $BASH_ENV
+
+definitions:
+  caches:
+    terminusplugins: ${TERMINUS_PLUGINS_DIR:-~/.terminus/plugins}
+
+# This is a larger yaml anchor, reused for each pipeline
+default_steps: &default_steps
+  - step:
+      name: Configure Environment Variables
+      services:
+        - docker
+      caches:
+        - docker
+        - terminusplugins
+      script:
+        - *bash_env_export
+        - export CI_BRANCH=${BITBUCKET_BRANCH:-master} && echo $CI_BRANCH
+        - export PR_NUMBER=$BITBUCKET_PR_ID && echo $PR_NUMBER
+        - export CI_PR_URL=https://bitbucket.org/$BITBUCKET_REPO_OWNER/$BITBUCKET_REPO_SLUG/pull-requests/$BITBUCKET_PR_ID && echo $CI_PR_URL
+        - export CI_BUILD_NUMBER=$BITBUCKET_BUILD_NUMBER && echo $CI_BUILD_NUMBER
+        - export CI_PROJECT_USERNAME=$BITBUCKET_REPO_OWNER && echo $CI_PROJECT_USERNAME
+        - export CI_PROJECT_REPONAME=$BITBUCKET_REPO_SLUG && echo $CI_PROJECT_REPONAME
+        - export CI_PROJECT_NAME=$BITBUCKET_REPO_FULL_NAME && echo $CI_PROJECT_NAME
+        - ./.ci/build/set-environment
+      artifacts:
+        - .bashrc
+  - parallel:
+      - step:
+          name: Static Tests
+          services:
+            - docker
+          caches:
+            - docker
+            - composer
+            - terminusplugins
+          script:
+            - *bash_env_export
+            - *bash_env_source
+            - ./.ci/test/static/run
+      - step:
+          name: Build PHP
+          services:
+            - docker
+          caches:
+            - docker
+            - composer
+            - terminusplugins
+          script:
+            - *bash_env_export
+            - *bash_env_source
+            - ./.ci/build/php
+          artifacts:
+            - web/**
+            - vendor/**
+  - step:
+      name: Deploy to Pantheon
+      services:
+        - docker
+      caches:
+        - docker
+        - terminusplugins
+      script:
+        - *bash_env_export
+        - *bash_env_source
+        - ./.ci/deploy/pantheon/dev-multidev
+      artifacts:
+        - web/**
+        - vendor/**
+        - drush/**
+        - console/**
+  - parallel:
```

Comment:
Any reason not to run Behat and Visual Regression in parallel @ataylorme ?

Reply:
Yes, Behat changes could make visual tests fail. For eaxample, a new post would change latest posts and archive templates. 

The current Behat tests in this example change site name and description, which are visible on all pages. If Backstop runs after Behat makes those updates the comparison of the multidev to dev will fail.

I would expect those who don't need/want tests, such as Behat, to remove them.